src/config/texts.js

```diff
@@ -9,6 +17,9 @@ export const texts = {
     pointsOfInterest: 'Orte',
     tours: 'Touren'
   },
+  errors: {
+    noData: 'Bitte überprüfen sie ihre Internetverbindung.'
```

Comment:
`sie ihre` -> `Sie Ihre`

Reply:
done in fc785872ca155350a23f1886ecfd91d39ae4d089